test/MultiFieldTests/MultiFieldCellFieldsTests.jl

```diff
@@ -116,6 +118,50 @@ cellmat1_Γ = integrate(((n⋅dv.⁺)-dq.⁻)*((n⋅du.⁺)+dp.⁻),quad_Γ)
 cellmat2_Γ = integrate((n⋅dv.⁺)*(n⋅du.⁺)+(n⋅dv.⁺)*dp.⁻-dq.⁻*(n⋅du.⁺)-dq.⁻*dp.⁻,quad_Γ)
 test_array(cellmat1_Γ,cellmat2_Γ,≈)
 
+# Test function evaluation
```

Comment:
As with the implementation, I would move the tests into `test/CellDataTests/CellFieldsTests.jl`